config.yml

```diff
@@ -32,46 +34,50 @@ trading_options:
     - DOWN
     - UP
 
-  # Maximum number of trade 'slots' at any time (your USDT balance must be at least MAX_COINS * QUANTITY)
-  MAX_COINS: 7
+  # Maximum number of trade 'slots' at any time (your USDT balance must be at least TRADE_SLOTS * QUANTITY)
+  TRADE_SLOTS: 999
 
   # the amount of time in MINUTES to calculate the difference from the current price (minimum: 1).
-  TIME_DIFFERENCE: 2
+  TIME_DIFFERENCE: 3
 
   # Number of times to check for TP/SL during each TIME_DIFFERENCE (minimum: 1).
   # Don't spam the Binance API, you will be banned (max 1200 requests per minute per IP).
-  RECHECK_INTERVAL: 10
+  RECHECK_INTERVAL: 36
 
   # the difference in % between the first and second checks for the price.
-  CHANGE_IN_PRICE: 10
+  CHANGE_IN_PRICE: 2
 
   # define in % when to sell a coin that's not making a profit.
-  STOP_LOSS: 5
+  STOP_LOSS: 99
 
   # define in % when to take profit on a profitable coin.
-  TAKE_PROFIT: .8
+  TAKE_PROFIT: .7
 
   # Use custom tickers.txt list for filtering pairs.
   CUSTOM_LIST: True
-   
+
   # Name of custom tickers list
-  TICKERS_LIST: 'tickers.txt'
-   
+  TICKERS_LIST: 'tickers_ETH.txt'
+
   # whether to use trailing stop loss or not; default is True
   USE_TRAILING_STOP_LOSS: True
 
   # when hit TAKE_PROFIT, move STOP_LOSS to TRAILING_STOP_LOSS percentage points below TAKE_PROFIT hence locking in profit
   # when hit TAKE_PROFIT, move TAKE_PROFIT up by TRAILING_TAKE_PROFIT percentage points
   TRAILING_STOP_LOSS: .4
   TRAILING_TAKE_PROFIT: .1
-  
+
   # Trading fee in % per trade.
   # If using 0.75% (using BNB for fees) you must have BNB in your account to cover trading fees.
   # If using BNB for fees, it MUST be enabled in your Binance 'Dashboard' page (checkbox).
   TRADING_FEE: .075
-  
+
+  # show/hide additional data on bot screen. False = silent, True = spam (debug mode).
+  VERBOSE_MODE: False
+
   SIGNALLING_MODULES:
-    - pausebotmod
-    - signalsamplemod
-    
+    #    - pausebotmod
```

Comment:
Dont turn off these modules

Reply:
have reset config values and have a separate config file now, thank you